nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-administration/pom.xml

```diff
@@ -86,5 +86,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-collections4</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Was this added intentionally? I can't see any tests in the `nifi-administration` module and if I remove this from the `pom.xml`, the module still builds successfully.